sublime_jedi/helper.py

```diff
@@ -33,28 +48,97 @@ def docstring_panel(view, docstring):
         sublime.status_message('Jedi: No results!')
 
 
-def docstring_tooltip(view, docstring, content_builder):
+def docstring_tooltip(view, docstring, location=None):
     """Show docstring in popup.
 
     :param view (sublime.View): current active view
     :param docstring (basestring): python __doc__ string
-    :param content_builder (callable): callable object should accept docstring
-        and return content read for popup (a html text)
+    :param location (int): The text point where to create the popup
     """
-    if docstring:
-        content = content_builder(docstring)
-        view.show_popup(content, max_width=512)
+    if not docstring:
+        return sublime.status_message('Jedi: No results!')
+
+    if location is None:
+        location = view.sel()[0].begin()
+
+    # fallback if mdpopups is not available
+    if not _HAVE_MDPOPUPS:
```

Comment:
@deathaxe restructured text is a part of default markup for python docs :) 
I think it possible to add support to `*.rst`, at one day, definitely not in the current PR.

But our code base should be extendable. Currently, not.

We can create an `Docstring Builder` *interface*(a shape of expected renderer) and pass current state (docstring, location, etc.) to it, and receive rendered docstring.

Then we can follow the contract provided by the interface, and build as much renders as we want (for `*.md`, `*.rst` or somethin else). And give possibility to users, specify the docstring format (like in PyCharm, for ex.).

I really like you idea to render `*.md` docstrings, but I think we can improve it and build a base for future features (:

PS: I can help